roles/openshift_web_console/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+openshift_web_console_nodeselector: {"region":"infra"}
```

Comment:
Unfortunately the default we use for this is defined in openshift_hosted role but we can redefine it here. This pattern would align it with other components while still allowing them to override it specifically for the console.

```
openshift_hosted_infra_selector: "region=infra"
openshift_web_console_nodeselector: "{{ openshift_hosted_infra_selector }}"
```
@michaelgugino agreed or is there a better way?